### PR TITLE
chore(docs): fix worflow badges on rendered README

### PR DESCRIPTION
**Pull request by jansegre**
- [Broken README from v0.52.1 release](https://github.com/HathorNetwork/hathor-core/blob/v0.52.1/README.md)
- [Fixed README from this PR](https://github.com/HathorNetwork/hathor-core/blob/chore/fix-workflow-badges/README.md)

This is related to https://github.com/badges/shields/issues/8671

## Acceptance Criteria

- badges for `tests` and `build` workflows should be rendered correctly now

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -81,7 +81,7 @@ jobs:
       if: startsWith(matrix.os, 'macos')
       run: |
         brew cleanup -q
-        brew update -q
+        # brew update -q
```
Comment (luislhl) [inline .github/workflows/main.yml:84]: Is this related to the badges problem, or just another unrelated fix?

**Review by luislhl — APPROVED** [review]

**Review by pedroferreira1 — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (jansegre) [reply .github/workflows/main.yml:84]: No, it's unrelated. This just fixes the CI on macOS.
